### ISSUE TITLE
Opening hidden files

### ISSUE THREAD

**Issue by csgillespie** [posit-dev/positron#3887]
## Positron Version:

Positron Version: 2024.06.1 build 27
Code - OSS Version: 1.90.0
Commit: a893e5b282612ccb2200102957ac38d3c14e5196
Date: 2024-06-26T01:22:29.024Z
Electron: 29.4.0
Chromium: 122.0.6261.156
Node.js: 20.9.0
V8: 12.2.281.27-electron.0
OS: Linux x64 6.5.0-41-generic

I'm also using regolith as a windows manager on Ubuntu

## Steps to reproduce the issue:

1. Open positron
2. Open file
3. Select the menu for hidden files (see screen shot)
 
![2024-07-05_09-10](https://github.com/posit-dev/positron/assets/1267747/b33398ea-72fe-4d36-a958-7bc2167a453d)

## Were there any error messages in the output or Developer Tools console?

Not that I could see

**Comment by lionel-** [comment]
@csgillespie What happened and what did you expect to happen?

**Comment by csgillespie** [comment]
The box in the bottom right hand corner is "odd" and can't be changed

**Comment by petetronic** [comment]
@csgillespie does this reproduce in your environment with VSCode or VSCodium?

**Comment by juliasilge** [comment]
We checked this out with some of the Linux setups that folks on our team use (such as Gnome) and were not able to see any problems; options for 'Show Hidden Files' work correctly in the file dialogs we checked, such as 'Open File'.

I'm going to close this for now because we suspect this must be specific to a certain window manager, but we definitely are interested in hearing about problems, especially any that you see _only_ in Positron and not in VS Code or VSCodium. Thank you for reporting the problem!
